app.py

```diff
@@ -1412,7 +1412,35 @@ def hour_of_code(level, program_id=None):
         ))
 
 
+def possibly_migrate_quizzes_parsons_tabs(customizations, parsons_hidden, quizzes_hidden):
```

Comment:
I see that this code is repeated in the For Teachers Module, perhaps we can make this code a function in that module and import it here?

Reply:
Yes indeed, just reused that function in the foreteachers!